### PR TITLE
[docs] Added changelog #25

### LINKED ISSUE #25: [docs] Add change log
Add `CHANGES.rst` as we have in the rest of OpenWISP modules. For the previous versions we can include a brief summary while we'll be more detailed with the new versions.

### PR DESCRIPTION
**Pull request by devkapilbansal**
Closes #25

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+- #9 [deps] Added support for django 1.9
+
+Version 0.2.2 [2015-06-16]
+--------------------------
+
+- [deps] Added support for django~=1.6.0
+- [deps] Added support for Python 3.3
+- [docs] Fix model reference in README
+- #2 [docs] Notes for load_model initialization
```
Comment (atb00ker) [inline CHANGES.rst:41]: Is this `#2` here intentional?

**Review by atb00ker — APPROVED** [review]

File: CHANGES.rst (hunk 1 above)
Comment (devkapilbansal) [reply CHANGES.rst:41]: Yes, I use #2 here because it was explained in that issue.

Comment (nemesisdesign) [reply CHANGES.rst:41]: can you change it to something like the following?
```
[docs] Notes for load_model initialization (`for more info see #2 <issue_url>`_)
```

File: CHANGES.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
+Changelog
+=========
+
+Version 0.1.0 [2020-01-15]
```
Comment (nemesisdesign) [inline CHANGES.rst:4]: you mean 1.1.2?

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Great work @devkapilbansal, this is very helpful!

See below.

File: CHANGES.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+- [deps] Drop python<3.3 support
+- [deps] Added support for python 3.7
+- [deps] Django 2 support added
+
+Version 1.1.0 [2017-05-11]
+--------------------------
+
+- [test] Added tests for swapper.split
+- #13 [fix] Handle contrib apps and apps with dot in app_label.
```
Comment (nemesisdesign) [inline CHANGES.rst:21]: same here and other occurrences

File: CHANGES.rst (hunk 2 above)
Comment (devkapilbansal) [reply CHANGES.rst]: Fixed now

**Comment by devkapilbansal** [comment]
Don't know why checks failed for python 3.5

**Comment by sheppard** [comment]
It looks like the latest setuptools_scm is using the f-string syntax introduced in Python 3.6.  One option would be to preinstall an older version of setuptools_scm for Python 3.5 tests.

On the other hand, Python 3.5 is EOL anyway so perhaps it can be dropped as well.

**Comment by nemesisdesign** [comment]
> It looks like the latest setuptools_scm is using the f-string syntax introduced in Python 3.6. One option would be to preinstall an older version of setuptools_scm for Python 3.5 tests.
> 
> On the other hand, Python 3.5 is EOL anyway so perhaps it can be dropped as well.

I'm in favor of just dropping it: https://github.com/openwisp/django-swappable-models/pull/30.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
@devkapilbansal please remove the commit that changes the build setting because there's a dedicated PR to that.

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**